### ISSUE TITLE
Reload import javascript without restarting gerbera

### ISSUE THREAD

**Issue by cweiske** [gerbera/gerbera#2234]
During import script development I have to restart gerbera whenever I change the javascript. This means I have to wait until gerbera scans all the files during startup, or that I have to change the autoscan configuration temporarily to speed up startup.

It would be nice if I could send some signal to gerbera to make it reload (or unload) the import javascript. That way the import script tinkering process would be:

1. Change import script
2. Send signal to gerbera with `killall --signal $number gerbera`
3. Test import with `touch /path/to/file.ogg`
4. Repeat until the script is fine

That's much faster than waiting for restarts.

Related: #2181

**Comment by KarlStraussberger** [comment]
Good point. Does not happen very often, but would be really helpful to test a script.

Could even be based on autoscan mechanisms (inotify, timed)

**Comment by cweiske** [comment]
Great!
